aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsReadmeDecorator.kt

```diff
@@ -62,7 +63,7 @@ class AwsReadmeDecorator : RustCodegenDecorator<ClientCodegenContext> {
 
                     ```toml
                     [dependencies]
-                    aws-config = "${codegenContext.settings.moduleVersion}"
+                    aws-config = "$awsConfigVersion"
```

Comment:
Exactly. The service crates get generated before the version manifest is created. This is OK though since service crate generation only needs the version numbers of runtime crates.